tests/benchmark/linpack2.c

```diff
@@ -241,11 +241,11 @@ double cpu_time ( void )
 
     The CPU time measurements available through this routine are often
     not very accurate.  In some cases, the accuracy is no better than
-    a hundredth of a second.  
```

Comment:
This is upstream code, please revert this file.

Reply:
Fixed